### ISSUE TITLE
AI in Azure portal: Show more details of events

### ISSUE THREAD

**Issue by Ruud-cb** [Microsoft/ApplicationInsights-Home#216]
Hi,

The events are a nice way of categorizing things, but unfortunately it does not show details as how the rest of the telemetry items.

![image](https://user-images.githubusercontent.com/10037514/39426229-548c753a-4c7f-11e8-9f36-f83e28644354.png)

Is there a way I can configure to always show the first Custom Data entry? Or is there a default name convention that will be shown below it?

![image](https://user-images.githubusercontent.com/10037514/39426260-71c2d572-4c7f-11e8-9ed1-4b5e810e7aec.png)

To make it look something like this:

![image](https://user-images.githubusercontent.com/10037514/39426406-f52ab74a-4c7f-11e8-9299-fd273c4ca69e.png)

If not then this is a future request, not sure if this is the right place though...
Currently I have to click each individual Event to show its content, kind of takes long if there are some duplicates.

**Comment by SergeyKanzhelev** [comment]
@zakimaksyutov do you think this feature can be implemented?

**Comment by zakimaksyutov** [comment]
@Ruud-cb, our recommendation is to use Analytics for such experiences. For instance,

```
customEvents
| where timestamp > ago(1d)
| top 101 by timestamp desc

```

Then you can use 'project', 'where' clauses to filter out events and what to show.

**Comment by SergeyKanzhelev** [comment]
@Ruud-cb if you think this UI feature is needed - please suggest it on http://aka.ms/aiuservoice.

@zakimaksyutov suggestion should work for scenario you asked. ([tabular operators](https://docs.loganalytics.io/docs/Language-Reference/Tabular-operators) `project`, and `extend` may help to make this experience even better).

**Comment by SergeyKanzhelev** [comment]
Please feel free to re-open issue if you need more asssistance

**Comment by Ruud-cb** [comment]
@zakimaksyutov Thanks for your feedback and suggestion. However this would not be user friendly, maybe it helps to explain how I work:
I am currently getting a lot of Events with the critical status because I enabled a generic error handler in the front-end, currently it spits out a lot of false-positives. Once I fixed/filtered it out I don't have to open them once again. Eventually if one serious occurs I use the "All available telemetry for this user session", (which is a really great feature btw!), this would not really work within Analytics, would need to switch back and forth.
I will defiantly add it as a suggestion on the link Sergey provided. Thanks for the quick response!
